src/main/java/com/minecolonies/client/render/RenderBipedCitizen.java

```diff
@@ -44,6 +48,7 @@
     public RenderBipedCitizen(RenderManager renderManagerIn)
     {
         super(renderManagerIn, defaultModelMale, 0.5F);
+        this.addLayer(new LayerBipedArmor(this));
```

Comment:
Yeah I plan on adding eventually in some future moment a real equipment GUI like the player one, but that's gonna be a lot of work and mix our current inventory code up.